pkg/scheduler/extender_test.go

```diff
@@ -388,3 +390,146 @@ func TestIsInterested(t *testing.T) {
 		})
 	}
 }
+
+func Test_convertToNodeNameToMetaVictims(t *testing.T) {
+	type args struct {
+		nodeNameToVictims map[string]*extenderv1.Victims
+	}
+	tests := []struct {
+		name string
+		args args
+		want map[string]*extenderv1.MetaVictims
+	}{
+		{
+			name: "test NumPDBViolations is transferred from nodeNameToVictims to nodeNameToMetaVictims",
+			args: args{
+				nodeNameToVictims: map[string]*extenderv1.Victims{
+					"node1": {
+						Pods: []*v1.Pod{
+							{ObjectMeta: metav1.ObjectMeta{Name: "pod1", UID: "uid1"}},
+							{ObjectMeta: metav1.ObjectMeta{Name: "pod3", UID: "uid3"}},
+						},
+						NumPDBViolations: 1,
+					},
+					"node2": {
+						Pods: []*v1.Pod{
+							{ObjectMeta: metav1.ObjectMeta{Name: "pod2", UID: "uid2"}},
+							{ObjectMeta: metav1.ObjectMeta{Name: "pod4", UID: "uid4"}},
+						},
+						NumPDBViolations: 2,
+					},
+				},
+			},
+			want: map[string]*extenderv1.MetaVictims{
+				"node1": {
+					Pods: []*extenderv1.MetaPod{
+						{UID: "uid1"},
+						{UID: "uid3"},
+					},
+					NumPDBViolations: 1,
+				},
+				"node2": {
+					Pods: []*extenderv1.MetaPod{
+						{UID: "uid2"},
+						{UID: "uid4"},
+					},
+					NumPDBViolations: 2,
+				},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := convertToNodeNameToMetaVictims(tt.args.nodeNameToVictims); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("convertToNodeNameToMetaVictims() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestHTTPExtender_convertToNodeNameToVictims(t *testing.T) {
+	type args struct {
+		nodeNameToMetaVictims map[string]*extenderv1.MetaVictims
+		nodeNames             []string
+		podsInNodeList        []*v1.Pod
+		nodeInfos             framework.NodeInfoLister
+	}
```

Comment:
ditto, no need for this type, just define the parameters directly in the test case struct below.

Reply:
hi @ahg-g thanks for your review. I have changed the test fun name and args according to your suggestions.

Reply:
there are still two more comments: change `convertToNodeNameToVictims` to `convertToVictims` and `convertToNodeNameToMetaVictims` to `convertToMetaVictims`

Reply:
@ahg-g OK, I have changed

Reply:
thanks, please squash the commits

Reply:
Hi @ahg-g, I have squashed the commits into one.